src/core/hoc/multiple-input.tsx

```diff
@@ -0,0 +1,41 @@
+import { common } from '../../translations/en.json';
+import { ControlledInput } from '../../ui/core/input';
+import { Text } from '../../ui/core/text';
+import { View } from '../../ui/core/view';
+
+type FieldProps = {
+  name: string;
+  label: string;
+  required?: boolean;
+  placeholder?: string;
+};
+
+type FromInputsProps = {
+  title: string;
+  fields: FieldProps[];
+  control: any;
+  editable: boolean;
+};
+
+export const FormInputs = ({ title, fields, control, editable }: FromInputsProps) => {
+  return (
+    <View>
+      <Text variant="h6-bold" className="my-4">
+        {title}
+      </Text>
+      {fields.map(({ label, name, placeholder, required }) => (
+        <View key={name} className="flex-1">
+          <Text variant="h6" className="mb-1.5">
+            {label} {required ? common.labels.starSign : ' '}
+          </Text>
+          <ControlledInput
+            control={control}
+            name={name}
+            placeholder={placeholder}
+            editable={editable}
+          />
+        </View>
+      ))}
+    </View>
+  );
+};
```

Comment:
Is it me or this component is not being used? Seems like the component should be the only the view with the text and controlled input, so it can be reused easily.

Reply:
Done.